snippets/gift-card-recipient-form.liquid

```diff
@@ -0,0 +1,164 @@
+{% comment %}
+  Renders gift card recipient form.
+  Accepts:
+  - product: {Object} product object.
+  - form: {Object} the product form object.
+  - section: {Object} section to which this snippet belongs.
+
+  Usage:
+  {% render 'gift-card-recipient-form', product: product, form: form, section: section %}
+{% endcomment %}
+
+<div class="customer">
+  {%- assign gift_card_recipient_control_flag = "properties[__shopify_send_gift_card_to_recipient]" -%}
+  <script src="{{ 'recipient-form.js' | asset_url }}" defer="defer"></script>
+  <recipient-form class="recipient-form"
+    data-section-id="{{ section.id }}"
+    data-product-variant-id="{{ product.selected_or_first_available_variant.id }}"
+  >
+    <input
+      id="Recipient-Checkbox-{{ section.id }}"
+      type="checkbox"
+      name="{{ gift_card_recipient_control_flag }}"
+      disabled
+      unchecked
+    >
+    <label class="recipient-checkbox" for="Recipient-Checkbox-{{ section.id }}">
+      <svg
+        width="1.6rem"
+        height="1.6rem"
+        viewBox="0 0 16 16"
+        aria-hidden="true"
+        focusable="false"
+      >
+        <rect width="16" height="16" stroke="currentColor" fill="none" stroke-width="1.75"></rect>
```

Comment:
I think this was right when using a `stroke-width` of `1`. Facet filters are using that as well. 
```suggestion
        <rect width="16" height="16" stroke="currentColor" fill="none" stroke-width="1"></rect>
```